tests/functional/jwt_helper.js

```diff
@@ -0,0 +1,85 @@
+const KJUR = require('jsrsasign')
+var fs = require('fs')
+const uuidV4 = require('uuid/v4');
+
+var jose = require('node-jose');
+var JWK = jose.JWK;
+var JWE = jose.JWE;
+
+var JSONWebKey = require( 'json-web-key' )
+
+const signingKey = '/Users/jonshaw/code/eq-survey-runner/tests/functional/sdc-user-authentication-signing-rrm-private-key.pem';
+const encryptionKey = '/Users/jonshaw/code/eq-survey-runner/tests/functional/sdc-user-authentication-encryption-sr-public-key.pem';
+
+export const generateToken = (schema, userId, collectionId, period_id, period_str, region_code, language_code, sexual_identity) => {
+    let signingKeyString = fs.readFileSync(signingKey, "utf8");  // get private key
+    let encryptionKeyString = fs.readFileSync(encryptionKey, "utf8");  // get private key
+
+    let eq_id = schema.substr(0,schema.indexOf('_'))
+    let form_type = schema.substr(schema.indexOf('_')+1).split('.', 1)[0]
```

Comment:
use a regex here, this is too fragile, look at the 'go' version for a regex to steal